HashRegistrarSimplified.sol

```diff
@@ -388,14 +388,21 @@ contract Registrar {
         entry h = _entries[_hash];
 
         h.value =  max(h.value, minPrice);
+        h.deed.setBalance(h.value, true);
```

Comment:
This should probably go in the 'else' clause - no point updating the balance twice if the if evaluates to true.